### PR TITLE
Update graphql.go

### PR DESCRIPTION
**Pull request by tvorogme**

### CODE REVIEW

**Review by wlofman021 — COMMENTED** [review]
trasction

File: services/dton/graphql.go [hunk 1]
```diff
@@ -56,7 +56,7 @@ func (m *Monitoring) GetMetrics(ctx context.Context) services.ApiMetrics {
 		Data struct {
 			Transactions []struct {
 				GenUtime string `json:"gen_utime"`
-			} `json:"transactions"`
+			} `json:"raw_transactions"`
```
Comment (wlofman021) [inline services/dton/graphql.go:59]: UQBJXDKkhFqZS2JvedNtLg9u6j32XkMPkHUc5cNSIMXFOQ3C

**Review by wlofman021 — COMMENTED** [review]
transaction

File: services/dton/graphql.go (hunk 1 above)
Comment (SupernLucky) [reply services/dton/graphql.go:59]: [](https://t.me/supernlucky)

**Pull request merged**